src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -101,12 +102,12 @@ export class ClrDatagridColumn extends DatagridFilterRegistrar<DatagridStringFil
 
     @ViewChild("columnHandle")
     set handleElRef(value: ElementRef) {
-        this._dragDispatcher.handleRef = value;
+        this._dragDispatcher.draggable.self = value.nativeElement;
     }
 
     @ViewChild("columnHandleTracker")
     set handleTrackerElRef(value: ElementRef) {
-        this._dragDispatcher.handleTrackerRef = value;
+        this._dragDispatcher.draggable.ghost = value.nativeElement;
```

Comment:
As mentioned before, calling `dragDispatcher.initialize(...)` in `ngAfterViewInit()` with all the necessary parameters would be cleaner than atomically mutating a bunch of "deep" properties all over the place.